### PR TITLE
766 UI problem

### LINKED ISSUE #766: UI problem when upgrading TestFlight
When I do a fresh TestFlight version there are no problems.

When I have a TestFlight old version and I click Upgrade to the latest one, it signs me out of my wallet and goes back to asking me to create a new one or restore one. Then I have the UI issue.

https://github.com/breez/c-breez/assets/127208515/9c112d0e-e79a-46d9-a889-4590280253d0

### PR DESCRIPTION
**Pull request by ubbabeck**
fixes #766

### CODE REVIEW

File: .github/workflows/build-ios.yml [hunk 1]
```diff
@@ -105,7 +105,7 @@ jobs:
         run: |
           rm -rf ../target
           cargo check
-          make ios-dev
+          make ios
```
Comment (erdemyerebasmaz) [inline .github/workflows/build-ios.yml:108]: !!! Good catch

File: lib/routes/initial_walkthrough/initial_walkthrough.dart [hunk 2]
```diff
@@ -51,88 +52,91 @@ class InitialWalkthroughPageState extends State<InitialWalkthroughPage>
...
-                      String frame = _animation!.value.toString().padLeft(2, '0');
-                      return Image.asset(
-                        'src/animations/welcome/frame_${frame}_delay-0.04s.png',
-                        gaplessPlayback: true,
-                        fit: BoxFit.cover,
-                      );
-                    },
+    return AnnotatedRegion<SystemUiOverlayStyle>(
+      value: themeData.appBarTheme.systemOverlayStyle!,
+      child: Scaffold(
```
Comment (erdemyerebasmaz) [inline lib/routes/initial_walkthrough/initial_walkthrough.dart:57]: We should wrap Scaffold's with Theme widgets that uses light theme, don't we?
```
Theme(
      data: breezLightTheme,
      child: Scaffold(...
```

**Review by erdemyerebasmaz — CHANGES_REQUESTED** [review]
Looks good!

I believe we're missing forcing light theme on start up widgets.

File: lib/routes/splash/splash_page.dart [hunk 3]
```diff
@@ -30,17 +31,21 @@ class SplashPageState extends State<SplashPage> {
...
-                fit: BoxFit.contain,
-                gaplessPlayback: true,
-                width: MediaQuery.of(context).size.width / 3,
-              ),
-            )
-          : const SizedBox(),
+    final themeData = Theme.of(context);
+    return AnnotatedRegion<SystemUiOverlayStyle>(
+      value: themeData.appBarTheme.systemOverlayStyle!,
+      child: Scaffold(
```
Comment (erdemyerebasmaz) [inline lib/routes/splash/splash_page.dart:37]: https://github.com/breez/c-breez/pull/769#discussion_r1464875886

File: .github/workflows/build-ios.yml (hunk 1 above)
Comment (ubbabeck) [reply .github/workflows/build-ios.yml:108]: Thanks 💯

File: lib/routes/initial_walkthrough/initial_walkthrough.dart (hunk 2 above)
Comment (ubbabeck) [reply lib/routes/initial_walkthrough/initial_walkthrough.dart]: True, this has been fixed.

**Pull request merged**